Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -443,6 +443,8 @@ public protocol Visitor {
 
   /// Called with the raw bytes that represent any unknown fields.
   mutating func visitUnknown(bytes: Data) throws
+    
+ func shouldIncludeDefault() -> Bool
```

Comment:
Needs documentation since it is public interface a `Visitor` needs to know about.